server/src/main/java/org/elasticsearch/search/retriever/KnnRetrieverBuilder.java

```diff
@@ -201,7 +201,7 @@ public RetrieverBuilder rewrite(QueryRewriteContext ctx) throws IOException {
     public QueryBuilder topDocsQuery() {
         assert queryVector != null : "query vector must be materialized at this point";
         assert rankDocs != null : "rankDocs should have been materialized by now";
-        var rankDocsQuery = new RankDocsQueryBuilder(rankDocs, null, true);
+        var rankDocsQuery = new RankDocsQueryBuilder(rankDocs, null, true, Float.MIN_VALUE);
```

Comment:
This should use a default member that you create and reference as suggested earlier. Alternately consider a convenience constructor that sets the default.